src/components/Overlay/__stories__/Overlay.scss

```diff
@@ -0,0 +1,31 @@
+@use '../../variables';
```

Comment:
Better to rename the file to `OverlayShowcase` to not confuse with the component's CSS file